gpg-interface.c

```diff
@@ -35,15 +36,27 @@ static const char *x509_sigs[] = {
 	NULL
 };
 
+static const char *ssh_verify_args[] = {
+	NULL
+};
+static const char *ssh_sigs[] = {
+	"-----BEGIN SSH SIGNATURE-----",
+	NULL
+};
+
 static struct gpg_format gpg_format[] = {
-	{ .name = "openpgp", .program = "gpg",
+	{ .name = "openpgp",
+	  .program = "gpg",
```

Comment:
Please do not mix in style changes with non-style changes. It makes it unnecessarily hard to review the code.